Frameworks/AEPSwiftUI/Sources/ContentCards/ContentCardCustomizable.swift

```diff
@@ -13,7 +13,7 @@
 import Foundation
 
 /// Protocol for customizing Content Card templates
-public protocol ContentCardCustomizer {
+public protocol ContentCardCustomizable {
```

Comment:
since this protocol is the actor not the object, i think we should call it `ContentCardCustomizing`.